recipes-core/system-api/system-api_v0.7.0.bb

```diff
@@ -10,8 +10,10 @@ INITSCRIPT_PARAMS = "defaults 81"
 
 GO_IMPORT = "github.com/flashbots/system-api"
 SRC_URI = "git://${GO_IMPORT};protocol=https;branch=main \
-           file://system-api-init"
-SRCREV = "v0.4.0"
+           file://system-api-init \
+           file://basic-auth-secret.mustache \
+           file://systemapi-config.toml.mustache"
```

Comment:
don't forget to add this to the render-config.bb in your other [PR](https://github.com/flashbots/meta-evm/pull/39#pullrequestreview-2458200721) to be able to render those new changes you added here too.